.drone.yml

```diff
@@ -17,7 +17,14 @@ steps:
     environment:
       GOPATH: /go
       GOBIN: /go/bin
-      IMG: autonomy/cluster-api-provider-talos:latest
+
+  - name: gen-manifests
+    image: golang:latest
```

Comment:
We have a container that has `gitmeta` already installed: https://github.com/talos-systems/talos/blob/master/.drone.yml#L29. Not sure if you need golang in the container. If you have to you can `go get github.com/talos-systems/talos`.